server/go/server.go

```diff
@@ -115,34 +115,33 @@ func handleCustomerPortal(w http.ResponseWriter, r *http.Request) {
 	}
 
 	var req struct {
-		CustomerID string `json:"customerId"`
+		SessionID string `json:"sessionId"`
 	}
 	if err := json.NewDecoder(r.Body).Decode(&req); err != nil {
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		log.Printf("json.NewDecoder.Decode: %v", err)
 		return
 	}
 
-	// This is the ID of the Stripe Customer and is typically stored in your
-	// database and would be retrieved along with the currently authenticated
-	// user. For demonstration, we're storing this value in the environment
-	// variables.
-	stripeCustomerID := req.CustomerID
+	// For demonstration purposes, we're using the Checkout session to retrieve the customer ID. 
+  // Typically this is stored alongside the authenticated user in your database. 
```

Comment:
No idea why this comment line is indented weird. I don't see it misaligned locally 🤔

Reply:
Oh. Oh no. Tabs vs spaces. 😩

Reply:
Fixed!

Reply:
in go, you can run `gofmt` and that'll format everything the same. Go has some special leading characters that aren't spaces. We usually use that everywhere except in the docs.